server/neptune/gateway/event/check_suite_handler.go

```diff
@@ -0,0 +1,88 @@
+package event
+
+import (
+	"context"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/core/config/valid"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/logging"
+	contextInternal "github.com/runatlantis/atlantis/server/neptune/context"
+	"github.com/runatlantis/atlantis/server/neptune/workflows"
+	"github.com/runatlantis/atlantis/server/vcs"
+	"github.com/runatlantis/atlantis/server/vcs/provider/github"
+)
+
+type CheckSuite struct {
+	Action            CheckRunAction
+	HeadSha           string
+	Repo              models.Repo
+	Sender            models.User
+	InstallationToken int64
+	Branch            string
+}
+
+type CheckSuiteHandler struct {
+	Logger            logging.Logger
+	RootConfigBuilder rootConfigBuilder
+	Scheduler         scheduler
+	DeploySignaler    deploySignaler
+}
+
+func (h *CheckSuiteHandler) Handle(ctx context.Context, event CheckSuite) error {
```

Comment:
I'm not clear on what the difference is between the check suite handler and check run handler, mind clarifying in comments or in the PR description?